core_impl/src/test/java/com/google/instrumentation/trace/SpanImplTest.java

```diff
@@ -221,6 +232,55 @@ public void toSpanData_EndedSpan() {
   }
 
   @Test
+  public void droppingAttributes() {
```

Comment:
It would be nice to also test the effect of reinsertion on the dropped attributes.